File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;


import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    //按条件,分页,查询
    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

    //品牌新增
    public Result insert(TbBrand tbBrand);

    //品牌修改
    public Result update(TbBrand tbBrand);

    //查询byid
    public TbBrand findById(Long id);

    //删除
    public void delete(Long[] ids);

    public List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    //按条件分页查询
    this.findPage=function (pageNum,pageSize,conditions) {
        return $http.post('../brand/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize,conditions);
    }

    //品牌新增
    this.insert=function (entity) {
        return $http.post("../brand/insert.do",entity);
    }
    //修改品牌信息
    this.update=function(entity){
        return $http.post("../brand/update.do",entity);
    }
    //查找单个
    this.findOne=function (id) {
        return $http.get("../brand/findById.do?id="+id);
    }
    //删除
    this.delete=function (ids) {
        return $http.get("../brand/delete.do?ids="+ids);
    }

    this.selectOptionList=function () {
        return $http.get("../brand/selectOptionList.do");
    }
})